src/Sentry.Unity/SystemInfoAdapter.cs

```diff
@@ -48,7 +49,7 @@ private SentrySystemInfoAdapter()
         public string? CpuDescription => SystemInfo.processorType;
         public string? DeviceName => SystemInfo.deviceName;
         public string? DeviceUniqueIdentifier => SystemInfo.deviceUniqueIdentifier;
-        public string? DeviceModel => SystemInfo.deviceModel;
+        public Lazy<string> DeviceModel => new(() => SystemInfo.deviceModel);
```

Comment:
Does it need to run in the UI thread?

Reply:
Docs don't say anything about it. 
Maybe we could make all values that don't require running in the UI thread become Lazy<T>?

Reply:
> Does it need to run in the UI thread?
I also haven't found any info about `SystemInfo.deviceModel` to be called on UI thread.

> Maybe we could make all values that don't require running in the UI thread become Lazy?
We can, but I need to revisit all the properties... again.

Reply:
> I also haven't found any info about SystemInfo.deviceModel to be called on UI thread.

Probably easy to test with `Task.Run(() => SystemInfo.deviceModel)` if that throws we know.

> We can, but I need to revisit all the properties... again)

Could be a new PR thingy, this solves already the main perf issue you've found right?

Reply:
> Could be a new PR thingy, this solves already the main perf issue you've found right?

Yes, when Sentry inits we no longer get `SystemInfo.deviceModel` call (~ 20ms), only when event is processed it's called and the result reused.